app/models/content_item.rb

```diff
@@ -2,6 +2,11 @@ class ContentItem
   include Mongoid::Document
   include Mongoid::Timestamps::Updated
 
+  def self.revert(previous_item:, item:)
+    item.remove unless previous_item
+    previous_item&.upsert
```

Comment:
The code used to be:

```
if previous_item
  previous_item.upsert
else
  item.remove
end
```

But Robocop complained that I should be using the `&.` (only calls the method if the operand is not `nil`) operating instead. Personally I think this code is less clear, but it was the only way to make Robocop happy.